cmd/manager/main.go

```diff
@@ -137,60 +148,13 @@ func main() {
 	}
 }
 
-// addMetrics will create the Services and Service Monitors to allow the operator export the metrics by using
```

Comment:
should we get rid of the reference to this function that is commented out on line 138-140?